.gitignore

```diff
@@ -43,9 +43,20 @@ local_test/*
 !local_test/index.html
 local_test/js/*
 local_test/css/*
+# folder of projects to test 
+local_test/intesting
```

Comment:
Do we need this in .gitignore? The local_test folder is already there for testing so I don't see the need for another subfolder. But I do think we should add an item to the PR checklist to make sure we revert the index.html file to its original state.